packages/datadog-instrumentations/test/multer.spec.js

```diff
@@ -15,7 +15,7 @@ withVersions('multer', 'multer', version => {
     })
 
     before((done) => {
-      const express = require('../../../versions/express').get()
+      const express = require('express')
```

Comment:
why this change ? shouldn't multer decide what version it's supposed to use from the external.json file thingy ?

Reply:
wouldn't that just use the express we have in our package.json ?

Reply:
_sheepish admission_ i don't know how to use external.json correctly. What needs to change here to make that happen?

Note: This multer test was _not actually running at all_ prior to this PR. That's part of what's being corrected.

Reply:
I also am not 100% how to use external.json, BUT from what I understand, if a plugin/instrumentation test needs to depend on another library for the setup, it needs to be put in the externals.json like in my PR here: https://github.com/DataDog/dd-trace-js/pull/4670/files#diff-c79dfa6aa60b124902dd80d1f3f70cfaea8caa71b801e93d7cc23b38d1e4527d

Reply:
at least this is how I understand the external.json file, but it's always been blurry for me tbh

Reply:
I changed this so it's now in externals and referring to it via `versioons`.